lib/src/generic/interceptors/csrf_interceptor.dart

```diff
@@ -58,9 +60,18 @@ class CsrfInterceptor extends Interceptor {
   Future<Context> onOutgoing(Provider provider, Context context) async {
     // Inject CSRF token into headers.
     if (context is HttpContext) {
-      if (!context.request.headers.containsKey(_header)) {
+      if (context.request.headers.containsKey(_header)) {
+        if (context.request.headers[_header] == null) {
+          throw new ArgumentError('CSRF header value can not be null');
+        }
+        ;
```

Comment:
not sure what you mean, there shouldn't need to be a semicolon there at all, right?